File: ui/src/app/model/UATDashboardVO.ts
export class UATDashboardVO {
    inProgress: number;
    completed: number;
    notStarted: number;
    total: number;

    constructor() {
        this.inProgress = -1;
        this.completed = -1;
        this.notStarted = -1;
        this.total = -1;
    }
}

File: domain/src/main/java/com/centram/common/vo/UATDashboardVO.java
package com.centram.common.vo;


import lombok.*;

import java.util.Set;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class UATDashboardVO {
    private Long inProgress;
    private Long completed;
    private Long notStarted;
    private Long total;
}